### PR TITLE
add remote and branch options to gh-pages

### PR DESCRIPTION
**Pull request by vision-ken**
for able to config  remote and branch options to gh-pages

### CODE REVIEW

File: packages/bisheng/src/index.js [hunk 1]
```diff
@@ -230,11 +232,11 @@ exports.deploy = function deploy(program) {
   if (program.pushOnly) {
     const output = typeof program.pushOnly === 'string' ? program.pushOnly : './_site';
     const basePath = path.join(process.cwd(), output);
-    pushToGhPages(basePath);
+    pushToGhPages(basePath, program);
```
Comment (benjycui) [inline packages/bisheng/src/index.js]: This will be better:

```jsx
pushToGhPages(basePath, {
  remote: program.remote,
  branch: program.branch,
});
```

**Comment by benjycui** [comment]
Thx

**Comment by benjycui** [comment]
Waiting for https://github.com/benjycui/bisheng/pull/89

**Comment by vision-ken** [comment]
Thx too.

**Comment by benjycui** [comment]
0.24.2

**Pull request merged**